.github/workflows/build-native.yml

```diff
@@ -0,0 +1,17 @@
+name: Build tt-native
+
+on: [pull_request]
+
+jobs:
+  build-tt-console:
+    runs-on: ubuntu-22.04
+    name: Build tt-console
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Run the script
```

Comment:
Can we upload an artifact here so that those who are lazy (me) don't have to build the console?

Reply:
I think we'll just deploy this on all CI machines as a static binary. It may change at any time (e.g. after TLB kernel changes are incorporated), so it's probably best not to rely on a precompiled binary for now.